symengine/cwrapper.cpp

```diff
@@ -489,6 +489,47 @@ char* ascii_art_str()
     return cc;
 }
 
+
+void ntheory_gcd(basic s, const basic a, const basic b)
+{
+    s->m = SymEngine::gcd(static_cast<const Integer &>(*(a->m)), static_cast<const Integer &>(*(b->m)));
```

Comment:
@isuruf Should this be refactored? https://github.com/symengine/symengine/pull/852/files#diff-72450588351eec10a19415c0a981ad41R177 Maybe in another PR.